### PR TITLE
OCPBUGS-8441: UPSTREAM <carry>: Update goutils dependency to v1.1.1 (#1140)

### PR DESCRIPTION
**Pull request by dougsland**
master commid-id: 0372c67802c9d12933d36dd54a7013d0d6ec31ae

Signed-off-by: Petr Horacek <hrck@protonmail.com>
Signed-off-by: Douglas Schilling Landgraf <dougsland@redhat.com>

### CODE REVIEW

**Comment by dougsland** [comment]
@cybertron 4.12 backport.

File: vendor/modules.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ github.com/Azure/go-ansiterm/winterm
 # github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
 ## explicit
 github.com/MakeNowJust/heredoc
-# github.com/Masterminds/goutils v1.1.0
+# github.com/Masterminds/goutils v1.1.1 => github.com/Masterminds/goutils v1.1.1
```
Comment (mkowalski) [inline vendor/modules.txt:11]: This looks fun because before and after `=>` you have the same stuff, but if that works, that works :)

**Pull request merged**